File: Bellini/DataAccessLayer/Models/Question.cs
using System.Text.Json.Serialization;

namespace DataAccessLayer.Models
{
    public class Question : BaseModel
    {
        public string Text { get; set; } = null!;
        public int GameId { get; set; }
        [JsonIgnore] public Game Game { get; set; } = null!;
        public bool IsCustom { get; set; }
        public List<AnswerOption> AnswerOptions { get; set; } = new List<AnswerOption>();
    }
}

File: Bellini/client.react/src/utils/interfaces/Question.ts
import {Game} from "@/utils/interfaces/Game.ts";
import {AnswerOption} from "@/utils/interfaces/AnswerOption.ts";

export interface Question {
    id: number;
    text: string;
    gameId: number;
    game: Game;
    isCustom: boolean;
    answerOptions: AnswerOption[];
}
